### PR TITLE
Fix wrong download URL, closes #25

### LINKED ISSUE #25: Download site down?
The download link been mentioned in readme: http://dl.gitea.io.de/lgtm
I got this error in chrome: `ERR_NAME_NOT_RESOLVED`, nslookup can't find the result of dl.gitea.io.de

### PR DESCRIPTION
**Pull request by metalmatze**
This was just a wrong URL and should be fixed.

### CODE REVIEW

**Comment by strk** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ including LGTM (looks good to me) in their approval text.
 ## Install
 
 You can download prebuilt binaries from the GitHub releases or from our
-[download site](http://dl.gitea.io.de/lgtm). You are a Mac user? Just take
+[download site](https://dl.gitea.io/lgtm). You are a Mac user? Just take
```
Comment (andreynering) [inline README.md:19]: Shouldn't it be `https://dl.gitea.io/gitea`?

Comment (tboerger) [reply README.md:19]: It's for lgtm ;)

**Comment by tboerger** [comment]
LGTM even if we will drop it soon

**Comment by strk** [comment]
Also, should it be "Are you a Mac user?"

**Pull request merged**